src/pages/VoucherRedemption/StoreBanner.tsx

```diff
@@ -66,6 +59,7 @@ const Header = styled.div`
   line-height: 30px;
 `;
 const OwnerImage = styled.img`
+  object-fit: cover;
```

Comment:
Added this because the merchant logo was getting squished. Looks like a lot of our merchant logos are longer horizontally than they are vertically, so they were getting squished w/o `object-fit: cover`. We could also use contain or the default, but cover seemed the best to me (although there are some risks of the restaurant name being cut off in cover):

Cover
<img width="1676" alt="Screen Shot 2020-10-05 at 4 58 43 PM" src="https://user-images.githubusercontent.com/10658691/95147221-f8df2700-0734-11eb-8363-8d557799c238.png">

Contain
<img width="1671" alt="Screen Shot 2020-10-05 at 5 56 17 PM" src="https://user-images.githubusercontent.com/10658691/95147236-02688f00-0735-11eb-8f35-f70fbeb51b72.png">

Default
<img width="1676" alt="Screen Shot 2020-10-05 at 5 32 09 PM" src="https://user-images.githubusercontent.com/10658691/95147229-fe3c7180-0734-11eb-9f47-d261bacb4ff2.png">

Reply:
never mind probably don't need to get any input since we're using scl logo across the board here

Reply:
The SCL logo is just the fallback right? Cover looks like the right option here, I think your instinct is right! The only time I imagine that could be weird is if we have the logo as a photo of the owners--but I do think we've generally tried to avoid that anyway.